### ISSUE TITLE
Chiseled Bookshelf missing?

### ISSUE THREAD

**Issue by ILikeOrangeJuice** [SkriptLang/Skript#6432]
### Skript/Server Version

```markdown
[00:08:08 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[00:08:08 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[00:08:08 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[00:08:08 INFO]: [Skript] Server Version: git-Paper-193 (MC: 1.20.1)
[00:08:08 INFO]: [Skript] Skript Version: 2.8.2 (skriptlang-github)
[00:08:08 INFO]: [Skript] Installed Skript Addons: 
[00:08:08 INFO]: [Skript]  - skript-placeholders v1.5 (https://github.com/APickledWalrus/skript-placeholders)
[00:08:08 INFO]: [Skript]  - skript-yaml v1.4.1-for-MC-1.20
[00:08:08 INFO]: [Skript]  - skript-reflect v2.4 (https://github.com/SkriptLang/skript-reflect)
[00:08:08 INFO]: [Skript]  - SkBee v3.2.1 (https://github.com/ShaneBeee/SkBee)
[00:08:08 INFO]: [Skript] Installed dependencies: 
[00:08:08 INFO]: [Skript]  - Vault v1.7.3-b131
[00:08:08 INFO]: [Skript]  - WorldGuard v7.0.9-beta1+2249-223b80c
```

### Bug Description

Using the line "on rightclick on chiseled_bookshelf" results in the error  "Can't understand this structure: on rightclick on chiseled bookshelf"

Using the line split up like this:
on rightclick on any block:
	if event-block is chiseled bookshelf:
it does work fine.
In the Discord it got said it is eventually a missing alias.

### Expected Behavior

It recognizing the first variant.

### Steps to Reproduce

Use the code as mentioned aboved in a script.

### Errors or Screenshots

"Can't understand this structure: on rightclick on chiseled bookshelf" 

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by ShaneBeee** [comment]
> Using the line "on rightclick on chiseled_bookshelf" 

Have you tried without the underscore?
Skript currently does not support block data in click events, and aliases do not have underscores.
You said `if event-block is chiseled bookshelf:` works fine, which tells me the alias is in fact there.

Even if the alias were missing, Skript auto-generates missing aliases. 
This tells me that the issue is due to the underscore you have in the event.

**Comment by ILikeOrangeJuice** [comment]
Hey, to confirm I tried each variant again:

chiseled_bookshelf:
[02:27:03 INFO]: [Skript] Can't find the script 'spawnprotection.sk' in the scripts folder!
[02:27:06 INFO]: [Skript] Reloading spawnprotections.sk...
[02:27:06 INFO]: Line 7: (spawnprotections.sk)
[02:27:06 INFO]:     Can't understand this structure: on rightclick on chiseled_bookshelf
[02:27:06 INFO]:     Line: on rightclick on chiseled_bookshelf:
[02:27:06 INFO]:  
[02:27:06 INFO]: [Skript] Encountered 1 error while reloading spawnprotections.sk! (18ms)

chiseled bookshelf:
seems to work this time, yesterday it did not, as seen in the error I posted previously in discord aswell:
[23:50:52 INFO]:     Can't understand this structure: on rightclick on chiseled bookshelf
[23:50:52 INFO]:     Line: on rightclick on chiseled bookshelf:

But trying it again now seemed to work, maybe a quirks and a restart fixed it I suppose? Either way seems to be solved, so I close this, thanks tho!:)
